python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -999,9 +1009,14 @@ class `wcsfit.FoFClass`, associating them into matches as you go.
                 goodInds = selected.selected & goodShapes
 
                 isStar = np.ones(goodInds.sum())
-                extensionIndex = np.flatnonzero(
+                findExtension = np.flatnonzero(
                     (extensionInfo.visit == visit) & (extensionInfo.detector == detector)
-                )[0]
+                )
+                if len(findExtension) == 0:
+                    # This extension does not have information necessary for
+                    # fit. Skip these detections.
```

Comment:
I'm not following this comment (I don't know what an extension is in this context). By "these detections", do you mean "this detector/visit"?

Reply:
"extension" is the `gbdes` term for one detector from one visit, and it's used throughout the `gbdes` task. I can change this line to "Skip the detections from this detector for this visit" to make it more clear.